### PR TITLE
apifctl edits per LS

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/apifctl-cicd-integration.md [hunk 1]
```diff
@@ -345,42 +319,106 @@ Specifies your tunnel ID for running tests using Sauce Connect Proxy.
...
+
+```bash
+-t product,production
+```
+
+---
+### `-tag <tag(s)>`
+<p><small>| STRING |</small></p>
+
+Adds a set of tags to the resulting event. Format as a comma-separated list of tags you want to assign to the test. For use with **[`run-tag`](#run-tag)** command only.
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:368]: -tag: the command (run-tag) will run all the tests in the project marked with that tag (e.g. - tag production: will run all the tests in the project that have "production" tag)

File: docs/api-testing/apifctl-cicd-integration.md [hunk 2]
```diff
@@ -345,42 +319,106 @@ Specifies your tunnel ID for running tests using Sauce Connect Proxy.
...
+
+```bash
+-t 2021-10-31T15:00
+```
+
+---
+### `-t <tag(s)>`
+<p><small>| STRING |</small></p>
+
+Adds a set of tags to the resulting event. Format as a comma-separated list of tags you want to assign to the test. For use with **[`exec`](#exec)** and **[`upload`](#upload)** commands.
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:358]: Replace with "Adds a set of tags to the test. Format as a comma-separated list of tags you want to assign to the test. For use with **[`exec`](#exec)** and **[`upload`](#upload)** commands."

File: docs/api-testing/apifctl-cicd-integration.md [hunk 3]
```diff
@@ -345,42 +319,106 @@ Specifies your tunnel ID for running tests using Sauce Connect Proxy.
...
 
-Identifies the maximum number of metrics to be shown in the list. Default value is `100`.
+Identifies the maximum number of metrics and events to be shown in the list. Default value is `100`. For use with **[`events`](#events)** and **[`metrics`](#metrics)** commands.
 
 ---
 ### `-o <offset>`
 <p><small>| INTEGER |</small></p>
 
-Specifies the number of events to be skipped from the beginning of the list. Default value is `0`.
+Specifies the number of events to be skipped from the beginning of the list. Default value is `0`. For use with **[`events`](#events)** and **[`metrics`](#metrics)** commands.
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:416]: Specifies the number of events and metrics...

**Review by LoryApiFortress — COMMENTED** [review]
Here's my further note for the lines where I cannot add a comment:
- **upload** command (line 149) the correct option is -t (not -tag) and it represents the tag(s) you want to assign to the test. More tags can be added as a comma-separated list

File: docs/api-testing/apifctl-cicd-integration.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ To utilize most `apifctl` functionalities, you'll need to add a webhook to your
    <img src={useBaseUrl('img/api-fortress/2021/04/createHook.png')} alt="Create New WebHook"/>
 3. Enter a name for your webhook (description is optional), then click **Save**.<br/>
    <img src={useBaseUrl('img/api-fortress/2021/04/sampleHook.png')} alt="sample webhook details" width="500" />
-4. The generated **Hook URL** will then appear. Your Sauce Labs credentials, the Sauce Labs REST API endpoint, and the `{hookId}` will populate automatically. It be formatted like this:
+4. The generated **Hook URL** will then appear. Your Sauce Labs credentials, the Sauce Labs REST API endpoint, and the `{hookId}` will populate automatically. It will be formatted like this:
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/api-testing/apifctl-cicd-integration.md:31]: @LoryApiFortress I left a comment about this here https://saucedev.atlassian.net/browse/TCD-1989?focusedCommentId=328927

**Comment by kimsauce** [comment]
> Here's my further note for the lines where I cannot add a comment:
> 
> * **upload** command (line 149) the correct option is -t (not -tag) and it represents the tag(s) you want to assign to the test. More tags can be added as a comma-separated list

done!

**Pull request merged**